sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -361,6 +361,28 @@ instances:
         #
         # collection_interval: 10
 
+    ## Configure collection of database schemas
+    #
+    ## schemas_collection
```

Comment:
looks like you need to sync these models

Reply:
you mean with example.yaml ?